Doc/library/decimal.rst

```diff
@@ -571,29 +585,10 @@ Decimal objects
       >>> Decimal(321).exp()
       Decimal('2.561702493119680037517373933E+139')
 
-   .. method:: from_float(f)
+   .. classmethod:: from_float(f)
 
-      Classmethod that converts a float to a decimal number, exactly.
-
-      Note `Decimal.from_float(0.1)` is not the same as `Decimal('0.1')`.
```

Comment:
Why did you drop the explanatory text?  The section is short so there is room for it.  Also, this text isn't incidental to the method; instead, it is entire reason have the method.

Please restore this text.